Source: Timo Gurr <tgurr@exherbo.org>
Upstream: Wait until the CMake conversion settled a bit, there are already open pull requests regarding doc handling
Reason: Don't try to install nonexistent files

diff -Naur doxygen-1.8.10/doc/CMakeLists.txt doxygen-1.8.10.new/doc/CMakeLists.txt
--- doxygen-1.8.10/doc/CMakeLists.txt	2015-06-20 23:49:50.000000000 +0200
+++ doxygen-1.8.10.new/doc/CMakeLists.txt	2015-07-01 18:15:02.607056864 +0200
@@ -88,17 +88,7 @@
         "${PROJECT_BINARY_DIR}/man/doxywizard.1"
         "${PROJECT_BINARY_DIR}/man/doxysearch.1"
         "${PROJECT_BINARY_DIR}/man/doxyindexer.1"
-        DESTINATION man/man1
-)
-
-install(FILES
-        "${PROJECT_BINARY_DIR}/latex/doxygen_manual.pdf"
-        DESTINATION "${CMAKE_INSTALL_PREFIX}/${DOC_INSTALL_DIR}"
-)
-
-install(DIRECTORY
-        "${PROJECT_BINARY_DIR}/html"
-        DESTINATION "${CMAKE_INSTALL_PREFIX}/${DOC_INSTALL_DIR}"
+        DESTINATION "${CMAKE_INSTALL_FULL_DATAROOTDIR}/man/man1"
 )
 
 endif()
